.gitattributes

```diff
@@ -67,3 +49,35 @@ Programs/test_frozenmain.h  linguist-generated=true
 *.html  diff=html
 *.py    diff=python
 *.md    diff=markdown
+
+# Generated files
+# https://github.com/github/linguist/blob/master/docs/overrides.md
+#
+# To always hide generated files in local diffs, mark them as binary:
+# $ git config diff.generated.binary true
+#
+[attr]generated linguist-generated=true diff=generated
+
+**/clinic/*.c.h                              generated
+*_db.h                                       generated
+Doc/library/token-list.inc                   generated
+Include/internal/pycore_ast.h                generated
+Include/internal/pycore_ast_state.h          generated
+Include/opcode.h                             generated
+Include/token.h                              generated
+Lib/keyword.py                               generated
+Lib/token.py                                 generated
+Modules/config.c                             generated
+Objects/typeslots.inc                        generated
+Parser/parser.c                              generated
+Parser/token.c                               generated
+Programs/test_frozenmain.h                   generated
+Python/Python-ast.c                          generated
+Python/deepfreeze/*.c                        generated
+Python/frozen_modules/*.h                    generated
+Python/frozen_modules/MANIFEST               generated
```

Comment:
These three lines are in `.gitignore`, I don't think we need them here.

Reply:
Yeah, I does not make sense to apply attributes to ignored files. Thanks.